lib/http-client.js

```diff
@@ -20,6 +20,9 @@ function responseError({
   } catch (e) {
     data = rawData;
   }
+  if (error.response === undefined) {
+    error.response = {};
```

Comment:
How about `error.response = error.response || {}`

Reply:
Sure :)

Reply:
Btw
https://github.com/DefinitelyTyped/DefinitelyTyped/pull/36046

Reply:
I wanted to mention, if you add a:
module.exports.default = module.exports
to the index.js, you'll support ES6 default import syntax.

If you did that, I could change the DefinitelyTyped library from:
export = Client
to
export default Client

While it might not seem like much, that allows the:
import Client from "fhir-kit-client"
syntax in ES6 and TypeScript without the "allowSyntheticDefaultImports" configuration for TypeScript.

It's totally up to you. It technically adds a circular reference on your exported function but unless something is doing something truly funky, it won't hurt and will make your package ES6 compatible once my PR is accepted.

Reply:
I also wouldn't mind a review by you of my type definitions as well.  I was able to use the existing FHIR types to greatly enhance the type specificity.  I was able to find a lot of defects in my services that use the library just because of this.

Reply:
> I wanted to mention, if you add a:
> module.exports.default = module.exports
> to the index.js, you'll support ES6 default import syntax.

Yea, this is part of a larger conversation we plan to have: https://github.com/Vermonster/fhir-kit-client/issues/110